using_images/other_images/jenkins.adoc

```diff
@@ -169,51 +162,109 @@ To create a new Jenkins application using standard Jenkins authentication:
 ----
 $ oc new-app -e \
     JENKINS_PASSWORD=<password> \
-    openshift/jenkins-1-centos7
+    openshift/jenkins-2-centos7
 ----
 
 [[jenkins-environment-variables]]
 === Environment Variables
 
 The Jenkins server can be configured with the following environment variables:
 
-.Jenkins Environment Variables
-[cols="4a,6a",options="header"]
-|===
+* `JENKINS_PASSWORD` (default: `password`)
++
+The password for the `admin` user when using standard Jenkins authentication.
+Not applicable when using {product-title} OAuth authentication.
 
-|Variable name |Description
+* `OPENSHIFT_ENABLE_OAUTH` (default: `false`)
++
+Determines whether the OpenShift Login plug-in manages authentication when
+logging into Jenkins. To enable, set to `true`.
 
-|`*JENKINS_PASSWORD*`
-|The password for the `*admin*` user when using standard Jenkins authentication.
-Not applicable when using {product-title} OAuth authentication.
+* `CONTAINER_HEAP_PERCENT` (default: `0.5`, i.e. 50%)
++
+Specifies the percentage of the container memory limit allocated to the Jenkins
+JVM maximum heap size (-Xmx).
+
+* `JENKINS_MAX_HEAP_UPPER_BOUND_MB`
++
+If set, specifies a static upper bound in MiB for the Jenkins JVM maximum heap
+size (-Xmx).  The effective setting used is the minimum of
+`CONTAINER_HEAP_PERCENT` and `JENKINS_MAX_HEAP_UPPER_BOUND_MB`. Example setting:
+`512`.
+
+* `JENKINS_JAVA_OPTIONS`
++
+Specifies additional options for the Jenkins JVM which may be overridden.
```

Comment:
I think this needs more details.  If the user sets this, they are replacing all the JVM options we auto-configured and the only JVM options Jenkins will get are whatever they set in this variable (meaning they must explicitly set max heap and anything else they care about):
https://github.com/openshift/jenkins/blob/master/2/contrib/s2i/run#L320-L322

Also not clear what "which may be overridden" means.  Overridden by who/what, and how are they overridden?